site/content/docs/user/known-issues.md

```diff
@@ -236,6 +236,22 @@ an 03 17:42:41 kind-1-control-plane kubelet[3804]: F0103 17:42:41.470269 3804 ku
 
 This problem seems to be related to a [bug in Docker][moby#9939].
 
+## Pod errors due to "too many open files"
```

Comment:
Please add an entry in the contents section https://github.com/kubernetes-sigs/kind/pull/917/files#diff-a43cc0c4cf108c28fe507969c9eaa365L19

Reply:
Done